system-tests/lib/crypto/csa.go

```diff
@@ -0,0 +1,12 @@
+package crypto
+
+import "strings"
+
+type CSAKey struct {
+	Key string
+}
+
+// CleansedKey returns the key without the "csa_" prefix
+func (c *CSAKey) CleansedKey() string {
```

Comment:
Are you using this anywhere? Would we ever want the value with the prefix? I suspect not, in which case let's move this to the boundaries of the system (the API where we get the CSA key from) and clean the value before it gets into the system